src/test/javascript/spec/component/exam/exam-update.component.spec.ts

```diff
@@ -184,6 +200,86 @@ describe('Exam Update Component', () => {
         expect(component.isSaving).toBeFalse();
     }));
 
+    it('should calculate the working time for RealExams correctly', () => {
+        exam.testExam = false;
+
+        exam.startDate = undefined;
+        exam.endDate = dayjs().add(2, 'hours');
+        fixture.detectChanges();
+        // Without a valid startDate, the workingTime should be 0
+        // exam.workingTime is stored in seconds
+        expect(exam.workingTime).toEqual(0);
+        // the component returns the workingTime in Minutes
+        expect(component.calculateWorkingTime).toEqual(0);
+
+        exam.startDate = dayjs().add(0, 'hours');
+        exam.endDate = dayjs().add(2, 'hours');
+        fixture.detectChanges();
+        expect(exam.workingTime).toBe(7200);
+        expect(component.calculateWorkingTime).toEqual(120);
```

Comment:
Both `toBe` and `toEqual` work to test against numbers. But it would make the code prettier if you would stick to one version. However, this is only a suggestion from my side.

Reply:
According to our testing guidelines, we should always use `toBe(..)` for primitive types like "number".